build/deployments/docker-compose.yaml

```diff
@@ -141,6 +318,64 @@ services:
     ports:
       - "5050:5050"
 
+  vm:
+    logging: *loki-logging
+    container_name: victoria-metrics
+    image: victoriametrics/victoria-metrics
+    restart: unless-stopped
+    ports:
+      - "8428:8428"
+    volumes:
+      - vm:/storage
+      - ./telemetry/victoria-metrics:/configs
+      - /var/run/docker.sock:/var/run/docker.sock
+    command:
+      - "--storageDataPath=/storage"
+      - "--httpListenAddr=:8428"
+      - "--promscrape.config=/configs/scrape.yml"
+
+  grafana:
+    logging: *loki-logging
+    container_name: grafana
+    image: grafana/grafana:8.5.2
+    depends_on:
+      - "vm"
+    environment:
+      # TODO(kdas): Figure out why including these plugins on some platforms causes this error:
+      #             `Error: ✗ Plugin not found (Grafana v8.5.2 linux-arm64)`
+      # - GF_INSTALL_PLUGINS="grafana-clock-panel,grafana-simple-json-datasource"
+      - GF_AUTH_DISABLE_LOGIN_FORM="true"
+      - GF_AUTH_ANONYMOUS_ENABLED="true"
+      - GF_AUTH_ANONYMOUS_ORG_NAME="Main Org."
+      - GF_AUTH_ANONYMOUS_ORG_ROLE="Admin"
+      - GF_USERS_ALLOW_SIGN_UP="false"
+    ports:
+      - "3000:3000"
+    volumes:
+      - grafanadata:/var/lib/grafana
+      - ./telemetry/grafana/provisioning/:/etc/grafana/provisioning/
+      - ./telemetry/dashboards/:/var/lib/grafana/dashboards/
+    restart: unless-stopped
+
+  loki:
+    # logging: *loki-logging
```

Comment:
@okdas Can we consult you on this? I remember I had issues with the loki driver so I commented it out. Let me know.

Reply:
This is not really required, we can keep this commented out in case we ever need to feed loki logs into loki. :) Or remove it, too. I understand there might be some dependencies issues (e.g. driver cannot push logs to loki until loki is available), though I did not have any issues loclaly.